src/main/java/teammates/common/datatransfer/FeedbackTextQuestionDetails.java

```diff
@@ -6,26 +6,35 @@
 import java.util.Map;
 
 import teammates.common.util.Const;
+import teammates.common.util.HttpRequestHelper;
 import teammates.common.util.Sanitizer;
 import teammates.common.util.Templates;
 import teammates.common.util.Templates.FeedbackQuestionFormTemplates;
 import teammates.ui.template.InstructorFeedbackResultsResponseRow;
 
 public class FeedbackTextQuestionDetails extends FeedbackQuestionDetails {
     
+    public int recommendedLength;
+
     public FeedbackTextQuestionDetails() {
         super(FeedbackQuestionType.TEXT);
+        this.recommendedLength = 0;
     }
     
     public FeedbackTextQuestionDetails(String questionText) {
         super(FeedbackQuestionType.TEXT, questionText);
+        this.recommendedLength = 0;
     }
 
     @Override
     public boolean extractQuestionDetails(
             Map<String, String[]> requestParameters,
             FeedbackQuestionType questionType) {
-        // Nothing to do here.
+        String recommendedLengthString = HttpRequestHelper.getValueFromParamMap(requestParameters,
+                Const.ParamsNames.FEEDBACK_QUESTION_TEXT_RECOMMENDEDLENGTH);
+
+        this.recommendedLength = recommendedLengthString == null || recommendedLengthString.isEmpty() ? 0
```

Comment:
As the `this` keyword is not necessary here, I would prefer to remove it (and at other parts where it is not necessary). This is also to maintain consistency as some `recommendedLength` are prefixed with `this` but some aren't.